src/request-idle-callback.mjs

```diff
@@ -18,10 +18,10 @@
 const requestIdleCallback = window.requestIdleCallback ||
   function (cb) {
     const start = Date.now();
-    return setTimeout(function () {
+    return setTimeout(() => {
```

Comment:
@addyosmani I don't suppose this breaks the fallback for older supported browsers?

If so, we should revert these changes changes.